build/yamls/kube-apiserver.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: v1
```

Comment:
I think we need to document this in a getting-started document and highlight our specific requirements. We don't need to provide a yaml file which is very cluster-specific and not applicable to most people.